sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -92,3 +92,8 @@ select abs(-3.13), abs('-2.19');
 
 -- positive/negative
 select positive('-1.11'), positive(-1.11), negative('-1.11'), negative(-1.11);
+
+-- width_bucket
+select width_bucket(5.35, 0.024, 10.06, 5);
+select width_bucket(5.35, 0.024, 10.06, -5);
```

Comment:
add a case for wrong input type: `select width_bucket(5.35, 0.024, 10.06, 0.5);`